src/main/java/com/marklogic/batch/rdbms/MigrateRdbmsToMarkLogicJobConfig.java

```diff
@@ -0,0 +1,125 @@
+package com.marklogic.batch.rdbms;
+
+import com.marklogic.client.document.DocumentWriteOperation;
+import com.marklogic.client.ext.helper.DatabaseClientProvider;
+import com.marklogic.spring.batch.columnmap.ColumnMapSerializer;
+import com.marklogic.spring.batch.columnmap.DefaultStaxColumnMapSerializer;
+import com.marklogic.spring.batch.columnmap.JacksonColumnMapSerializer;
+import com.marklogic.spring.batch.item.processor.ColumnMapProcessor;
+import com.marklogic.spring.batch.item.rdbms.AllTablesItemReader;
+import com.marklogic.spring.batch.item.writer.MarkLogicItemWriter;
+import com.marklogic.spring.batch.item.writer.support.DefaultUriTransformer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.batch.core.Job;
+import org.springframework.batch.core.Step;
+import org.springframework.batch.core.configuration.annotation.EnableBatchProcessing;
+import org.springframework.batch.core.configuration.annotation.JobBuilderFactory;
+import org.springframework.batch.core.configuration.annotation.JobScope;
+import org.springframework.batch.core.configuration.annotation.StepBuilderFactory;
+import org.springframework.batch.core.launch.support.RunIdIncrementer;
+import org.springframework.batch.item.ItemReader;
+import org.springframework.batch.item.database.JdbcCursorItemReader;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Import;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.jdbc.core.ColumnMapRowMapper;
+
+import java.util.Map;
+
+@EnableBatchProcessing
+@Import(value = {
+        ApplicationProperties.class,
+        com.marklogic.spring.batch.config.MarkLogicBatchConfiguration.class,
+        com.marklogic.spring.batch.config.MarkLogicConfiguration.class})
+@PropertySource(value = "classpath:application.properties", ignoreResourceNotFound = true)
+@PropertySource(value = "file:application.properties", ignoreResourceNotFound = true)
+public class MigrateRdbmsToMarkLogicJobConfig {
```

Comment:
This needs class comments too to explain how this fits into the puzzle. Most users of this won't be familiar enough with Spring Batch to know what a Config class is, or what all those annotations are doing.